config.yaml

```diff
@@ -8,7 +8,7 @@ common:
  http_proxy: ''
```

Comment:
I assume that we do not parse the endianess from the arch name and thus we do not require the endianess anywhere in our code? Otherwise, it would not support an architecture name without endianess (eg 'ppc64') and the old name would be better.